addons/medical_gui/functions/fnc_updateCategories.sqf

```diff
@@ -35,6 +35,30 @@ params ["_display"];
     _ctrl ctrlSetTextColor _color;
     _color set [-1, 0.8]; // Mouseover change
     _ctrl ctrlSetActiveColor _color;
+
+    private _actionsCount = count GVAR(possibleActions);
+    private _pageCount = ceil (_actionsCount/9);
```

Comment:
The page size should be moved to define as it's used in multiple places.

Reply:
Resolved in latest commit